charts/prometheus-node-exporter/templates/service.yaml

```
@@ -14,6 +14,7 @@ spec:
 {{- if .Values.service.ipDualStack.enabled }}
   ipFamilies: {{ toYaml .Values.service.ipDualStack.ipFamilies | nindent 4 }}
   ipFamilyPolicy: {{ .Values.service.ipDualStack.ipFamilyPolicy }}
+  externalTrafficPolicy: {{ .Values.service.externalTrafficPolicy }}
 {{- end }}
```

The field should not be affected by the conditional on dual stack.

I'd suggest to consider the value only if set by the user, i.e. let us keep it unset by default. 

```suggestion
{{- end }}
{{- if .Values.service.externalTrafficPolicy }}
  externalTrafficPolicy: {{ .Values.service.externalTrafficPolicy }}
{{- end }}
```